samples/transactional-session/cosmosdb/sample.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Using Transactional Session with CosmosDB
+reviewed: 2022-07-20
+component: Core
+related:
+- nservicebus/messaging/transactionalsession
+---
+
+This sample uses the transactional session package with the CosmosDB persistence package to achieve transactionally consistent database changes and message operations.
+
+downloadbutton
```

Comment:
Is this a thing? 😮

Reply:
apparently, that was copied from other sample doc pages 😄